Gruntfile.js

```diff
@@ -96,7 +96,9 @@ grunt.initConfig({
 			"test/autostart.html",
 			"test/startError.html",
 			"test/logs.html",
-			"test/setTimeout.html"
+			"test/setTimeout.html",
+			"test/reporter-html-legacy-markup.html",
+			"test/reporter-html-no-qunit-element.html"
```

Comment:
Yeh I noticed this too, tried to make it fail with an error but got lost :( A proper failure would be better than timing out. I assume it's because the error is being suppressed somewhere?